libraries/stdlib/genmsl/lib/mx_math.metal

```diff
@@ -21,6 +21,12 @@ T1 mx_mod(T1 x, T2 y)
     return x - y * floor(x/y);
 }
 
+template <typename T>
+T mx_inversesqrt(T x)
+{
+    return ::rsqrt(x);
+}
+
```

Comment:
Aren't we going to need a corresponding definition of mx_inversesqrt() in genglsl/lib/mx_math.glsl?

Reply:
most definitely, thanks for the catch - I was staring at metal code all day and had a brain fart.